s3transfer/futures.py

```diff
@@ -78,8 +78,9 @@ def cancel(self):
 
 class TransferMeta(object):
     """Holds metadata about the TransferFuture"""
-    def __init__(self, call_args=None):
+    def __init__(self, call_args=None, id=None):
```

Comment:
This shadow's the [built-in id](https://docs.python.org/3/library/functions.html#id).